pkg/autoscaler/autoscaler.go

```diff
@@ -71,6 +71,7 @@ func (r *Autoscaler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 }
 
 func (a *Autoscaler) Start() {
+	a.LeaderElection.AfterOnStoppedLeading(a.Deployments.StopScalers)
```

Comment:
cleanup desired scale and downscale when instance is not leader anymore

Reply:
I think this would break scale from 0 in the proxy which should still be possible

Reply:
I think this would break the `h.Deployments.AtLeastOne(deploy)` call in the proxies that are not a leader.

Reply:
nvm it won't break that since it's a different flow. Currently all stop does is to stop the scale down timer.